tools/test_prereqs.php

```diff
@@ -14,22 +17,70 @@
 $db = MyPDO::instance();
 
 $delete = Delete::new($db);
-$delete->from('Announcements')->perform();
-$delete->from('Authentication')->perform();
-$delete->from('MeetingAttendance')->perform();
-$delete->from('OfficialMeetings')->perform();
-$delete->from('ConComList')->perform();
-$delete->from('Registrations')->perform();
-$delete->from('BadgeTypes')->perform();
-$delete->from('VolunteerHours')->perform();
-$delete->from('HourRedemptions')->perform();
-$delete->from('Events')->perform();
-$delete->from('AnnualCycles')->perform();
-$delete->from('AccountConfiguration')->perform();
-$delete->from('Members')->perform();
-$delete->from('HourRedemptions')->perform();
-$delete->from('VolunteerHours')->perform();
-$delete->from('VolunteerRewards')->perform();
+try {
+    $delete->from('Announcements')->perform();
+} catch (Exception $e) {
+}
```

Comment:
I know this is just a useful tool for us, but I wonder if we could make a function and call it instead of using lots of try/catch blocks like this. I'm not sure if we can build functions outside of classes in PHP, the way we can with something like JS.

It seems like this is just cleaning up data within tables, is that correct?

```
private function __cleanupTable($table): void
{
  try {
      $delete->from($table)->perform();
  } catch (Exception $e) {
  }
}
```

And then we could even put our table names in an array and just `foreach` on it to delete them all with fewer lines of code.

Let me know if this makes sense to do, and if it doesn't that's also OK to leave this as-is.